README.md

```diff
@@ -156,6 +156,14 @@ dont_set_title = false
 # Sets the maximum amount of log files in the Logs folder (Default: 10). Equivalent to the '--melonloader.maxlogs' launch option
 max_logs = 10
 
+[mono_debug_server]
+# Let the Mono debug server wait until a debugger is attached when debug_mode is true (only for Mono games). Equivalent to the '--melonloader.debugsuspend' launch option
+debug_suspend = false
+# The IP address the Mono debug server will listen to when debug_mode is true (only for Mono games). Equivalent to the '--melonloader.debugipaddress' launch option
+debug_ip_address = "127.0.0.1"
+# The port the Mono debug server will listen to when debug_mode is true (only for Mono games). Equivalent to the '--melonloader.debugport' launch option
+debug_port = 10000
```

Comment:
Minor oversight, but the default value defined in the code is `55555`. @HerpDerpinstine maybe do a quick commit fixing this?

Reply:
oh, whoops, yeah I only decided later to use 55555 and forgot to change it, here https://github.com/LavaGang/MelonLoader/pull/865